src/frontend/components/UI/TopNavBar/index.module.scss

```diff
@@ -73,3 +73,8 @@
     display: none;
   }
 }
+
+.metaMaskIcon {
+  width: var(--space-md);
+  height: var(--space-md);
```

Comment:
This will go from 16 to 14px at the 1300px breakpoint. Is that what you intended?

I think it's fine to just hardcode here or use `--space-md-fixed` if you really want to use a value from the spacing system.

It also might be best to just set the width here. I think it is being slightly distorted by setting both since the svg aspect ratio isn't 1:1